doc/src/index.md

```diff
@@ -78,6 +92,8 @@ hand, while one can -- and usually must -- annotate types for the compiler, type
 compile time and cannot be manipulated or expressed at run time. In Julia, types are themselves
 run-time objects, and can also be used to convey information to the compiler.
 
+### [What Makes Julia, Julia?](@id man-what-makes-julia)
+
```

Comment:
Suggest changing this to be more specific: "Julia's type system and multiple dispatch" 
Also, placing the heading before the previous paragraph starting "Although one sometimes speaks..."

Reply:
This is possible, but I do think that the way it reads now is okay. The previous paragraph is still somewhat talking about how Julia is different from other languages, I personally like the way it reads right now.

Reply:
Ok, fair enough.